pyshipping/binpack.py

```diff
@@ -8,7 +8,7 @@
 """
 
 
-import binpack_simple
+import pyshipping.binpack_simple
```

Comment:
I have this error when calling the funtion `binpack` below this line
> NameError: name 'binpack_simple' is not defined

In my opinion, we should do something like this:
`import pyshipping.binpack_simple as binpack_simple`